cli/print/cluster_nodegroups.go

```diff
@@ -16,14 +16,14 @@ var nodeGroupsTmplSrc = `ID	NAME	DEFAULT	INSTANCE TYPE	NODES	DISK
 
 var nodeGroupsTmpl = template.Must(template.New("nodegroups").Parse(nodeGroupsTmplSrc))
 
-func NodeGroups(outputFormat string, w *tabwriter.Writer, addOns []*types.NodeGroup) error {
+func NodeGroups(outputFormat string, w *tabwriter.Writer, addons []*types.NodeGroup) error {
```

Comment:
The name should not be `addons`, but `nodegroups`

Reply:
Updated